website/content/en/docs/about/eks-compatibility.md

```diff
@@ -6,17 +6,21 @@ weight = 30
 
 ## Compatibility
 
-Amazon EKS maintains end-to-end testing between EKS Kubernetes versions and Kubeflow versions. The following table relates compatibility between Kubernetes versions on Amazon EKS and Kubeflow v1.5.
+Amazon EKS maintains end-to-end testing between EKS Kubernetes versions and Kubeflow versions. The following table relates compatibility between Kubernetes versions on Amazon EKS and Kubeflow v1.6.
 
 <div class="table-responsive">
   <table class="table table-bordered">
     <thead class="thead-light">
       <tr>
         <th>EKS Versions</th>
-        <th>Kubeflow v1.5</th>
+        <th>Kubeflow v1.6</th>
       </tr>
     </thead>
     <tbody>
+    <tr>
```

Comment:
make only 1.22 compatible for now, we will test others and add
we know that 1.21 and below doesnt work